### ISSUE TITLE
Switch to classic - iOS - The survey is skipped after tapping "Switch to Expensify classic"

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#51526]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**  9.0.54-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Issue reported by:** Applause - Internal Team

**Issue found when executing PR** https://github.com/Expensify/App/pull/51376
## Action Performed:
1. Open New.Expensify hybrid mobile app
2. Sign in to the account
3. Navigate to Settings tab
4. Tap "Switch to Expensify Classic"

## Expected Result:
A survey opens asking for a reason to switch to classic app after tapping "Switch to Expensify classic"

## Actual Result:
The survey is skipped, user redirected to the classic app

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details> 

https://github.com/user-attachments/assets/44030887-e01c-4103-bc18-7a1d9b68fe8c

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by lanitochka17** [comment]
@strepanier03  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by strepanier03** [comment]
I'll test this tomorrow, running out of steam today.

**Comment by war-in** [comment]
Hi 👋 
The entire discussion can be found in the PR linked above ⬆️ 
Also, [this](https://github.com/Expensify/App/pull/47856#issuecomment-2310429239) is the comment where we decided to remove the survey from the HybridApp

cc @AndrewGable since he has a context on internal discussions 🙏

**Comment by AndrewGable** [comment]
Yes we decided to remove this survey and it's expected behavior. @danielrvidal and @anmurali are discussing a cross platform solution in the #convert room and will open up a new issue to handle this, so we can close this one.
